gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -1165,6 +1165,9 @@ impl GatewayLightning for ClnRpcService {
                 tonic::Status::internal(e.to_string())
             })?
             .map_err(|e| tonic::Status::internal(e.to_string()))?;
+        let inbound_lightning_liquidity_msats = channels.clone().iter().fold(0, |acc, channel| {
+            acc + (channel.amount_msat.msat() - channel.our_amount_msat.msat())
+        });
```

Comment:
* Why clone here? `.iter()` takes a reference to `self`, so that cloned version will be dropped right away.
* Why use `fold` instead of `map` and `sum`?
* You aren't taking into account the channels reserve afaik. Instead of calculating the total amount receivable yourself you could use `receivable_msat` returned by `listpeerchannels` (it's an additional API call, but more likely to give us a correct picture. thoughts @m1sterc001guy @tvolk131?).

Reply:
Your third point makes sense. @mubarak23 sorry for the misdirection! You'll want to call this RPC: https://docs.corelightning.org/reference/lightning-listpeerchannels

Reply:
@mubarak23 any update on this? Would love to get it in soon

Reply:
@m1sterc001guy yes, i work on this but am have some issue with `nix develop ` command